SU-HSDO/suhumsci
# shs-128: fixes bug related to auto blocks and floated images

## Summary
This fixes a floated image bug on autogenerated blocks for the events page.

## Steps to Test
1. Go to `/events/colloquium-series` and observe the float is no longer collapsing and looking all messed up.

## Screenshots
![Screen Shot 2023-02-20 at 10 56 26 AM](https://user-images.githubusercontent.com/8405274/220159640-0cb57a4e-df96-4694-a23b-d684ecfdd8de.png)

## Notes
This came from a bug I noticed while doing work for 128 (image styles).
